packages/react-composites/src/composites/localization/locales/de-DE/strings.json

```diff
@@ -45,6 +45,8 @@
   "callWithChat": {
     "peopleButtonLabel": "Personen",
     "chatButtonLabel": "Chat",
+    "moreDrawerMicrophoneMenuTitle": "Microphone",

```

Comment:
Hmm looks like we're not stripping out beta feature strings (or at least the meting composite ones) -- assume that's OK?

Reply:
I don't think we can. We add en-US strings directly yo JSON file and there is no way to conditionally compile those files (IMO)

Reply:
Yeah don't think so either cc @PorterNan for awareness